rcaracaus/desarol
# 177: Removing Josefine Sans from header links

Looked into other fonts but I just decided to go with the font we're already using. If you don't end up liking the look of it another font I tried that looked okay was Lato.

Steps to test:
1) Go to any page
2) see header
